public/stylesheets/app/_case-progress-card.scss

```diff
@@ -60,8 +60,7 @@
   }
 
   .govuk-summary-list__row:last-child,
-  .govuk-summary-list__row:last-child > *,
-  .govuk-table__row:last-child .govuk-table__cell {
+  .govuk-summary-list__row:last-child > * {
```

Comment:
I think I understand what you are saying here - the component below ( > Add a note) should be a part of the table? Rather than an element below? Then the css would apply the no border to the last row (the now contained Add a note component?)

Reply:
Yes, I think that goes seamlessly with everything else existing.
Small correction to my comment that should go after line 26 (the for loop closing)